### PR TITLE
Only load additional emails when requested

### PR DESCRIPTION
**Pull request by chriscct7**
Part of #6124

### CODE REVIEW

File: includes/class-edd-customer.php [hunk 1]
```diff
@@ -699,6 +703,22 @@ public function decrease_value( $value = 0.00 ) {
...
+	 *
+	 * @since  x.y.z
+	 * @return array           The emails of the customer.
+	 */
+	public function get_emails() {
+		if ( array_key_exists( 'emails', $this->data  ) ) {
+            		return $this->data['emails'];
+        	} else {
+			$this->data['emails']   = (array) $this->get_meta( 'additional_email', false );
+			$this->data['emails'][] = $this->email;
```
Comment (cklosowski) [inline includes/class-edd-customer.php]: While we're in here, one thing that's always annoyed me is that the primary email is the `last` email in the array...can we make it so `$this->email` is the first value in the array of emails returned? It makes more sense for that to be first than last.

Comment (chriscct7) [reply includes/class-edd-customer.php]: @cklosowski It's been updated to support this

Comment (cklosowski) [reply includes/class-edd-customer.php]: 👍

**Comment by chriscct7** [comment]
@cklosowski just an fyi the since docbloc wasn't updated on this one for the new function

**Pull request merged**